internal/util/invalid_serialization_detector.go

```diff
@@ -90,15 +90,21 @@ func (d *InvalidSerializationDetectorSchema) SerializeType(data string) (any, er
 }
 
 // ApplyScope is for applying a scope to the references. Does not apply to this object.
-func (d InvalidSerializationDetectorSchema) ApplyScope(_ schema.Scope) {}
+func (d *InvalidSerializationDetectorSchema) ApplyScope(_ schema.Scope, _ string) {}
+
+// ValidateReferences validates that all necessary references from scopes have been applied.
+// Does not apply to this object.
+func (d *InvalidSerializationDetectorSchema) ValidateReferences() error {
+	return nil
+}
```

Comment:
Actually, it's funny that you have function comments here explaining what these (don't) do, but nothing for the other `ApplyScope` and `ValidateReferences` implementations which do something...

Reply:
Sure; but it's still more useful, for humans, to have comments where they say something useful about what it does. Hey, whatever: I won't hold this up.

Reply:
For the most part, the interface itself has enough documentation for most things.